spec/u-namespace.md

```diff
@@ -12,11 +12,14 @@ manages the specification for this namespace, hence the _namespace_ `u:`.
 
 ## Options
 
-This section describes common **_<dfn>`u:` options</dfn>_** which each implementation SHOULD support
+This section describes common **_<dfn>`u:` options</dfn>_** that apply
 for all _functions_ and _markup_.
```

Comment:
I don't like this phrasing, as it changes the meaning, not just the normativity.

```suggestion
This section describes **_<dfn>`u:` options</dfn>_**.
When implemented, they apply to all _functions_ and _markup_,
including user-defined _functions_ in that implementation.
```